app/src/auth.py

```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint, render_template, redirect, url_for, request, flash
 from .models import User, db
 from flask_login import login_user
-
+from passwordStrength import check_password_strength
```

Comment:
Add a `.` for local file imports

Reply:
Added a . before the file name passwordStrength.